editor/components/inserter/index.js

```diff
@@ -12,6 +12,7 @@ import IconButton from 'components/icon-button';
 class Inserter extends wp.element.Component {
 	constructor() {
 		super( ...arguments );
+		this.nodes = {};
```

Comment:
Hmm, doesn't work if you remove it.  Maybe I will change bindRefNode() and see if that changes things.